portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -208,6 +208,18 @@ definition {
 		Button.clickDone();
 	}
 
+	macro configureSubmissions {
+		AssertClick(
+			locator1 = "Form#SETTINGS_TAB_SUBMISSIONS",
+			value1 = "Submissions");
+
+		Click.javaScriptClick(
+			checkboxName = "Submission",
+			locator1 = "Checkbox#ANY_CHECKBOX");
```

Comment:
oi Natanel!

Sobre essa macro, como ela se tratar de setar uma configuracao especifica, acredito que o nome deveria ser diferente. Por exemplo: **configureOneSubmissionPerUser**. Deixando assim mais facil de entender o que essa macro faz! :)

Sobre a variavel _checkboxName_, e ineteressante que ela aponte para a checkbox que ela esta clicando, entao inves de apenas "Submissions" deve ser **"Limit to One Submission per User"**!